kong/plugins/syslog/schema.lua

```diff
@@ -6,6 +6,16 @@ local severity = {
   one_of = { "debug", "info", "notice", "warning", "err", "crit", "alert", "emerg" },
 }
 
+local facility = {
+	type = "string",
+	default = "FACILITY_USER",
+	one_of = { "FACILITY_AUTH", "FACILITY_AUTHPRIV",  "FACILITY_CRON",  "FACILITY_DAEMON",
+			"FACILITY_FTP",  "FACILITY_KERN",  "FACILITY_LPR",  "FACILITY_MAIL",
+			"FACILITY_NEWS", "FACILITY_SYSLOG",  "FACILITY_USER",  "FACILITY_UUCP",
+			"FACILITY_LOCAL0",  "FACILITY_LOCAL1", "FACILITY_LOCAL2",  "FACILITY_LOCAL3",
+			"FACILITY_LOCAL4",  "FACILITY_LOCAL5",  "FACILITY_LOCAL6",  "FACILITY_LOCAL7" },
+}
+
```

Comment:
Since the entity is called facility, the entries themselves can drop the prefix imo

Reply:
Sure.
I did this quite "mechanically", just for our own  purpose, so it's a bit rough, and probably should be refactored in some way.

Reply:
Do i have to do the changes myself and resend a PR ?

Reply:
OK, done :)

Reply:
can you please remove the tabs and use spaces?

Reply:
Sure , done.
Btw, i also removed in the previous commit the FACILITY_ prefixes, as requested.
Best regards.